src/util/roundtopixel.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <cmath>
+
+inline auto makeRoundToPixel(float devicePixelRatio) {
```

Comment:
```suggestion
inline auto makeRoundToPixelFnc(float devicePixelRatio) {
```

Reply:
An alternative would be to repalce auto: 
```suggestion
#include <functional>
inline std::function<float(float)> makeRoundToPixel(float devicePixelRatio) {
```

Reply:
yeah, this would accomplish type-erasure which is not needed here as well as a heap allocation (for the captured value)... just use the lambda with auto...